src/App.js

```diff
@@ -29,19 +30,17 @@ class App extends Component<Props> {
       <Router>
         <div className="app-container">
           <NavBar routes={routesData.routes} />
-
-          <Route exact path="/" component={() => <Home />} />
-
-          <Route path="/about" component={() => <About />} />
-
-          <Route
-            path="/members"
-            component={() => <Members membersData={membersData} />}
-          />
-
-          <Route path="/contact" component={() => <Contact />} />
-
-          <Route path="/candidates" component={() => <Candidates />} />
+          <Switch>
+            <Route exact path="/" component={() => <Home routes={routesData.routes} />} />
+            <Route path="/about" component={() => <About routes={routesData.routes}/>} />
+            <Route
+              path="/members"
+              component={() => <Members membersData={membersData} routes={routesData.routes} />}
+            />
+            <Route path="/contact" component={() => <Contact routes={routesData.routes}/>} />
+            <Route path="/candidates" component={() => <Candidates routes={routesData.routes}/>} />
+            <Redirect to="/"/>
+          </Switch>
```

Comment:
Do you think you can add the footer to the App.js instead of having it on every other component?

Reply:
Just as Navbar is

Reply:
yeah, we discussed this and it messed up the design, dont know why.  Like literally adding footer under switch and removing footer from every component. There was another option which was wrapping up the entire switch in a container and add the footer below. But it didnt work either since the mid part (container) shrinked 👎🏽 There might be a way tough, I believe this change is necessary to avoid repeating code.

Reply:
Could you add a todo? something like:

// TODO: what you explained in your comment